frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/utils.ts

```diff
@@ -22,13 +24,14 @@ import {
   PipelineTaskNodeModel,
   BuilderNodeModelData,
   PipelineRunAfterNodeModelData,
+  FinallyTaskNodeModelData,
 } from './types';
 
-const createGenericNode: NodeCreatorSetup = (type, width?) => (name, data) => ({
+const createGenericNode: NodeCreatorSetup = (type, width?, height?) => (name, data) => ({
   id: name,
   data,
-  height: NODE_HEIGHT,
-  width: width != null ? width : NODE_WIDTH,
+  height: height ?? NODE_HEIGHT,
+  width: width ?? NODE_WIDTH,
```

Comment:
Technically still a change in logic... but I think it gets the spirit of what I was after.